packages/jsapi-utils/src/TableUtils.test.ts

```diff
@@ -1192,6 +1617,7 @@ describe('quick filter tests', () => {
       const column = makeFilterColumn();
 
       expect(TableUtils.makeQuickTextFilter(column, null)).toBe(null);
+      // @ts-expect-error text is generally a string
       expect(TableUtils.makeQuickTextFilter(column, undefined)).toBe(null);
```

Comment:
We should just remove the `null` case in `makeQuickTextFilter`, it never gets called with `null` (or `undefined`)